apiserver/facade/interface.go

```diff
@@ -23,8 +23,39 @@ type Facade interface{}
 // Factory is a callback used to create a Facade.
 type Factory func(Context) (Facade, error)
 
+// LeadershipContext represents all the leadership methods that can be performed
```

Comment:
I had this as "... describes factory methods for objects that deliver specific lease-related capabilities."